backend/.env

```
@@ -8,7 +1,26 @@
 EMAIL_FROM="aasanrishtecontact@gmail.com"
+
+# Application URL (update for production)
 APP_URL="http://localhost:3000"
+API_URL="http://localhost:8000"
+
+# Feature Flags
 PAID_MODE="false"
+
+# File Upload
+UPLOAD_DIR="uploads"
+MAX_FILE_SIZE=2097152  # 5MB
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MAX_FILE_SIZE key should go before the UPLOAD_DIR key

(UnorderedKey)